src/DataTypes/Simulation.jl

```diff
@@ -0,0 +1,48 @@
+import CurricularAnalytics: DegreePlan
+
+mutable struct Simulation
+    degreePlan::DegreePlan                  # The curriculum that is simulated
+    duration::Int                           # The number of terms the simulation runs for
+
+    predictionModel::Module                 # Module that implements the model for predicting student's performance in courses
+
+    numStudents::Int                        # The number of students in the simulation
+    enrolledStudents::Array{Student}        # Array of students that are enrolled
+    graduatedStudents::Array{Student}       # Array of students that have graduated
+    stopoutStudents::Array{Student}         # Array of students who stopped out
+
+    studentProgress::Array{Int}             # Indicates wheter students have passed each course
+
+    gradRate::Float64                       # Graduation rate at the end of the simulation
+    termGradRates::Array{Float64}           # Array of graduation rates at the end of the simulation
+    timeToDegree::Float64                   # Average number of semesters it takes to graduate students
+    stopoutRate::Float64                    # Stopout rate at the end of the simulation
+    termStopoutRates::Array{Float64}        # Array of stopout rates for each term
```

Comment:
snake case these variables 😄

Reply:
Done. 179c41190af4e3708a1c12d1494416deb9d7eb74

Have got used to writing camel case 😅, it should be all fixed now。

Reply:
No worries haha, it was what I was used to as well